src/main/java/nl/dtls/fairdatapoint/api/controller/metadata/catalog/CatalogController.java

```diff
@@ -115,7 +115,7 @@ public CatalogMetadata storeCatalogMetaData(final HttpServletRequest request, Ht
         LOGGER.info("Request to store catalog metadata with IRI {}", uri.toString());
 
         metadata.setUri(uri);
-        metadata.setParentURI(VALUEFACTORY.createIRI(instanceUrl + "/fdp"));
+        metadata.setParentURI(VALUEFACTORY.createIRI(instanceUrl + ""));
```

Comment:
Looks like `instanceUrl` is already a string, so there should not be a need to coerce it into one explicitly. Refactoring remnant?

Reply:
Yes, you are right. I forgot to remove it.